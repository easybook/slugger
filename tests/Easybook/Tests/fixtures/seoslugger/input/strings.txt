The price is 5.99€
The price is £5.99
The price is $5.99
The price is ¥5990
5.99€ = 5 euros & 99 cents
£5.99 = 5 pounds & 99 pence
$5.99 = 5 dollars & 99 cents
¥5990 = 5990 yen
5.99€ is higher than $5.99
lorem@ipsum.com
lorem@ipsum
lorem & ipsum
lorem&ipsum at ipsum.com
my username is @lorem
The review gave 5* to the place
10% of 10* is 1*
use lorem@ipsum.com to get a 10% discount
E = mc^2
The gravity of Earth is 9.81 m/s2
The #3 comment at @lorem = 10% of *&*