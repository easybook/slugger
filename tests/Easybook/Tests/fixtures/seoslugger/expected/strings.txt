the-price-is-5-euros-99-cents
the-price-is-5-pounds-99-pence
the-price-is-5-dollars-99-cents
the-price-is-5990-yen
5-euros-99-cents-equals-5-euros-and-99-cents
5-pounds-99-pence-equals-5-pounds-and-99-pence
5-dollars-99-cents-equals-5-dollars-and-99-cents
5990-yen-equals-5990-yen
5-euros-99-cents-is-higher-than-5-dollars-99-cents
lorem-at-ipsum-dot-com
lorem-at-ipsum
lorem-and-ipsum
lorem-and-ipsum-at-ipsum-dot-com
my-username-is-at-lorem
the-review-gave-5-star-to-the-place
10-percent-of-10-star-is-1-star
use-lorem-at-ipsum-dot-com-to-get-a-10-percent-discount
e-equals-mc2
the-gravity-of-earth-is-9-dot-81-m-s2
the-number-3-comment-at-at-lorem-equals-10-percent-of-star-and-star